DEPLOYING.md

```diff
@@ -27,6 +27,10 @@ sudo systemctl enable docker
 sudo docker info
 ```
 
+#### Extra dependencies
+
+- You will also need SMTP server and it's credentials for CKAN to work properly. This will not obstacle deployment, CKAN will be up and running, but won't be able sending Emails (Eg on password reset).
```

Comment:
* a SMTP server and its credentials
* won't be able to send emails (e.g. on password reset).